src/lib/y2partitioner/clients/main.rb

```diff
@@ -58,15 +59,15 @@ def self.run(allow_commit: true)
         end
       end
 
-      # Ask whether to proceed with changing the disks;
-      # or inform that we will not do it.
+      # Ask whether to proceed with changing the disks; or inform that we will not do it.
+      #
       # @return [Boolean] proceed
       def self.should_commit?(allow_commit)
         if allow_commit
-          q = "Modify the disks and potentially destroy your data?"
+          q = _("Modify the disks and potentially destroy your data?")
```

Comment:
Is this the same message we used to have in yast-storage? Just asking to make sure we do not promote some temporary message to be a final one, specially if we have a translated version already.

Reply:
Well, it seems the old Partitioner does not show a similar warning. Instead, it has a last step to summarize all the actions to perform.